### PR TITLE
[9.0][FIX]stock_auto_move: assign automatic procurement to only auto moves without proc.

### PR DESCRIPTION
**Pull request by zakiuu**
**Actual behavior:** With the `stock_auto_move` module, moves that are marked _automatic_ are processed automatically without the interaction of the users. In the mean time, those moves are assigned to an `automatic procurement` blindly. The author of the module has a reason doing so, and it has been explained in #180.
**The issue:** Procurement group have a strong _functional semantic_ (For example: when doing a reception in two steps, a procurement group is generated to tie up the 2 created picking for each one of the steps.).

This PR attends to fix this issue, by adding an _automatic procurement_ to only moves that aren't assigned to a procurement group.

### CODE REVIEW

**Comment by zakiuu** [comment]
This PR is based on #358

**Review by lmignon — APPROVED** [review]
@zakiuu changes in https://github.com/OCA/stock-logistics-workflow/pull/364/commits/a274769ce37bd5cfd477fc7dfbd1d65179a50ac1 LGTM... We must wait for the merge of #358

**Comment by gurneyalex** [comment]
@npiganeau can you check this PR?

File: stock_auto_move/models/stock_location_path.py [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 class StockLocationPath(models.Model):
 
     _inherit = 'stock.location.path'
 
+    auto_confirm = fields.Boolean(
+        help="If this option is selected, the generated moves will be "
+             "automatically processed as soon as the products are available. "
+             "This can be useful for situations with chained moves where we "
+             "do not want an operator action."
+    )
```
Comment (npiganeau) [inline stock_auto_move/models/stock_location_path.py:16]: Although I do understand the main subject of the PR and agree with it, I do not understand why you modify the `stock.location.path` code logic. It seems to me unrelated to the subject of this PR. Maybe it could be in another PR ?

File: stock_auto_move/views/stock_location_path.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <field name="model">stock.location.path</field>
+        <field name="inherit_id" ref="stock.stock_location_path_form"/>
+        <field name="arch" type="xml">
+            <field name="auto" position="after">
+                <field name="auto_confirm"/>
+            </field>
+        </field>
+    </record>
+
+</odoo>
```
Comment (npiganeau) [inline stock_auto_move/views/stock_location_path.xml:18]: Same remark here

File: stock_auto_move/models/stock_location_path.py (hunk 1 above)
Comment (zakiuu) [reply stock_auto_move/models/stock_location_path.py:16]: Hi @npiganeau thanks for the review, but this PR is based on #359  and that's the reason why you see those commits here. I had to do so because the migration of the module wasn't merged yet

Comment (npiganeau) [reply stock_auto_move/models/stock_location_path.py:16]: Ok, I understand.

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by Jortolsa-S73 — APPROVED** [review]
